README.md

```diff
@@ -1,70 +1,125 @@
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://reactjs.org/docs/react-dom-server.html">React Server</a></li>
+  </ul>
+</details>
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+### Key Features <a name="key-features"></a>
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+- **There will be more greetings**
+- **There will be more colors**
+- **There will be a button for creating a new greeting**
 
-### `npm test`
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
 
-### `npm run build`
+## 💻 Getting Started <a name="getting-started"></a>
+### Prerequisites
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+In order to run this project you need:
+```sh
+ gem install rails and react
+```
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
+### Setup
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+Clone this repository and this one [ReactFrontEnd](https://github.com/DiegoMero/hello-rails-back-end):
 
-### `npm run eject`
 
-**Note: this is a one-way operation. Once you `eject`, you can't go back!**
+```sh
+  cd folder
+  git clone git@github.com:DiegoMero/hello-react-front-end.git
+```
 
-If you aren't satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+### Install
 
-Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you're on your own.
+Install this project with:
 
-You don't have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn't feel obligated to use this feature. However we understand that this tool wouldn't be useful if you couldn't customize it when you are ready for it.
+```sh
+  cd hello-react-front-end
+  bundle install
+```
 
-## Learn More
+### Usage
 
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
+To run the project, execute the following command:
 
-To learn React, check out the [React documentation](https://reactjs.org/).
+```sh
+  ./bin/dev
+```
 
```

Comment:
- [ **OPTIONAL** ] **RECOMMENDED:** I think it would be better if you update the above command to `npm start` as it is the one used to run an react project